<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ivan_QtTwoWayCommunicationClass</class>
 <widget class="QMainWindow" name="Ivan_QtTwoWayCommunicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ivan_QtTwoWayCommunication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="mGridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="mLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="mShowDialogButton">
      <property name="text">
       <string>mostra dialog passando da dll</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="mFullScreenButton">
      <property name="text">
       <string>FullScreen</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ivan_qttwowaycommunication.qrc"/>
 </resources>
 <connections/>
</ui>
